src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -62,7 +62,7 @@
     @DataBoundConstructor public BourneShellScript(String script) {
         this.script = Util.fixNull(script);
     }
-    
+
```

Comment:
As a general rule, avoid whitespace changes in PRs so as to keep the diff minimal.